x-pack/legacy/plugins/apm/public/components/app/ErrorGroupOverview/List/__test__/List.test.tsx

```diff
@@ -38,9 +37,9 @@ describe('ErrorGroupOverview -> List', () => {
   it('should render empty state', () => {
     const storeState = {};
     const wrapper = mount(
-      <MemoryRouter>
+      <MockUrlParamsProvider>
```

Comment:
this change was necessary because `List` now throws if no `serviceName` is defined in `urlParams`